students/ryba_paula/lesson_13_objects_and_classes/shallow_compare.py

```diff
@@ -0,0 +1,25 @@
+from object_insp import Bunny
+
+
+def shallow_compare(obj1, obj2, attributes):
+    for key in attributes:
+        if getattr(obj1, key) != getattr(obj2, key):
+            return False
+        return True
+
+
+def main():
+    peter = Bunny("Peter", 7, "red")
+    fluffy = Bunny("Fluffy", 2.5, "blue")
+    fluffy2 = Bunny("Fluffy", 2.5, "blue")
+
+    attributes = ['name', 'age', 'colour']
+
+    print("Are {} and {} equal?".format(peter.name, fluffy.name),
```

Comment:
możesz się zapoznać z metodami `__str__()` i `__repr__()` w klasie, które służą do konwersji obiektu do stringa i wyświetlania go, dzięki czemu ten zapis skróci sie do:
`print("Are {} and {} equal?".format(peter, fluffy)...`

Reply:
oo super, dzięki :)